Added datalake verifier to RandomNodeOperationsTest

mmaslankaprv opened the pull request:
Adding verifiers will give us the coverage for the translation process correctness. 

<!--
See https://github.com/redpanda-data/redpanda/blob/dev/CONTRIBUTING.md#pull-request-body
for more details and examples of what is expected in a PR body.

Content in this top section is REQUIRED. Describe, in plain language, the motivation
behind the change (bug fix, feature, improvement) in this PR and how the included
commits address it.

Add the GitHub keyword `Fixes` to link to bug(s) this PR will fix, e.g.
  Fixes #ISSUE-NUMBER, Fixes #ISSUE-NUMBER, ...

If this PR is a backport, link to the original with `Backport of PR`, e.g.
  Backport of PR #PR-NUMBER
-->

## Backports Required

<!-- Checking at least one of the checkboxes is REQUIRED if this PR is not a backport. -->

- [ ] none - not a bug fix
- [ ] none - this is a backport
- [ ] none - issue does not exist in previous branches
- [ ] none - papercut/not impactful enough to backport
- [x] v24.3.x
- [ ] v24.2.x
- [ ] v24.1.x

## Release Notes

<!--
If the changes in this PR do not need to be mentioned in the release
notes, then don't add a sub-section and simply list `none`, e.g.

* none

Otherwise, adding a sub-section or `none` is REQUIRED if the PR is not a backport PR.
If this is a backport PR, adding contents to this section will override
the release notes section inherited from the original PR to dev.

Add one or more of the sub-sections with a short description bullet
point of the change, e.g.

### Bug Fixes

* Short description of the bug fix if this is a PR to `dev` branch.

### Features

* Short description of the feature. Explain how to configure.

### Improvements

* Short description of how this PR improves existing behavior.

-->
- none

mmaslankaprv committed:
tests: added iceberg verifiers to random_node_operations_test

Signed-off-by: Michał Maślanka <michal@redpanda.com>

mmaslankaprv committed:
tests: always enable tiered storage in random_node_operations_test

In order to aviod combinatorial explosion of the test cases we always
enable tiered storage in random node operations test.

Signed-off-by: Michał Maślanka <michal@redpanda.com>

vbotbuildovich commented:
<!--NEW FAILURES-->
<!--a3be72f8-839c-4242-8dcf-0df88c332137-->
<!--body-->
non flaky failures in https://buildkite.com/redpanda/redpanda/builds/59367#01939b5e-5aa3-4d2f-b3e3-cc1cb3f4215a:
```
"rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.S3"
```
<!--/body-->

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/59367-->
<!--970f1f95-de4c-43e5-b6d8-f7641d5da466-->
<!--title-->
## Retry command for [Build#59367](https://buildkite.com/redpanda/redpanda/builds/59367)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```
/ci-repeat 1
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":1,"enable_failures":true,"mixed_versions":false,"with_iceberg":true}
```
<!--/body-->

mmaslankaprv commented:
/ci-repeat 1

dotnwat commented:
/ci-repeat 5
release
skip-units
dt-repeat=50
tests/rptest/tests/random_node_operations_test.py

dotnwat commented:
/ci-repeat 5
release
skip-units
dt-repeat=50
tests/rptest/tests/random_node_operations_test.py

mmaslankaprv commented:
/ci-repeat 5
release
skip-units
dt-repeat=2
tests/rptest/tests/random_node_operations_test.py

mmaslankaprv commented:
@dotnwat the `dt-repeat=50` wont work as this test is long and with 50 repeats it always times out

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/59719-->
<!--9b73676f-c805-471e-a90a-90dbd963e626-->
<!--3dccbabf-0724-4754-9598-fb03ea459cc9-->
<!--67450db3-af04-44cd-900d-c9c78226b9d5-->
<!--b9cf10ac-ddc2-4ce6-be7d-f6172fdd32ec-->
<!--6b360e1c-24c7-4bad-9569-e56ca88f0ec5-->
<!--b3de5129-2275-4dd9-b98b-72b33bbd8531-->
<!--29d49826-8fb6-4bf4-b523-e3b1fc3d690c-->
<!--05ebdae8-5c12-4c14-87b6-98f262955cd8-->
<!--8ef86868-9abd-4812-a086-ca0b3f3a97a3-->
<!--e97a7a7a-224a-43b7-b49e-ba2a3ba68636-->
<!--a32cf35a-5893-49f3-9a08-bff778d5d710-->
<!--046be2a0-458c-4c2f-907b-63c4b6b39b3f-->
<!--a2c193df-fbaf-4555-b2ef-b21f9b1ee44a-->
<!--d8b7b8bd-99f0-4f10-9ce4-740f5ecf2a8d-->
<!--title-->
## Retry command for [Build#59719](https://buildkite.com/redpanda/redpanda/builds/59719)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```


/ci-repeat 1
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":2,"enable_failures":true,"mixed_versions":true,"with_iceberg":false}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":2,"enable_failures":false,"mixed_versions":false,"with_iceberg":true}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":2,"enable_failures":true,"mixed_versions":false,"with_iceberg":true}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":2,"enable_failures":true,"mixed_versions":false,"with_iceberg":false}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":2,"enable_failures":false,"mixed_versions":true,"with_iceberg":false}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":2,"enable_failures":false,"mixed_versions":false,"with_iceberg":false}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":2,"enable_failures":true,"mixed_versions":true,"with_iceberg":true}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":1,"enable_failures":true,"mixed_versions":false,"with_iceberg":true}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":1,"enable_failures":false,"mixed_versions":false,"with_iceberg":true}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":1,"enable_failures":true,"mixed_versions":true,"with_iceberg":true}
```
<!--/body-->

vbotbuildovich commented:
<!--TEST RESULTS-->
<!--eacf9b20-2ed5-486f-8eb0-ab7818c9e52b-->
<!--TEST RESULTS-->
<!--1a0432fd-4ed7-4ea3-a000-cc0c4796e94f-->
<!--TEST RESULTS-->
<!--875628df-e332-43a7-996d-f469703209c0-->
<!--TEST RESULTS-->
<!--5b126bbe-7d53-4a02-9e7c-3fc2c57935df-->
<!--TEST RESULTS-->
<!--56fb5836-7684-4b9b-a38c-6204265ba7d4-->
<!--TEST RESULTS-->
<!--70a73ea4-3ef0-4ede-a83a-5e3a2b8066ac-->
<!--TEST RESULTS-->
<!--135626c0-375e-4cfa-b5a6-454fc51f9d2f-->
<!--TEST RESULTS-->
<!--ed72eca2-2a98-4eaf-8414-01fe480ccda3-->
<!--TEST RESULTS-->
<!--c81b672e-aac0-4068-9dba-b3a50096e913-->
<!--TEST RESULTS-->
<!--9deb5707-573b-4a6d-b810-0388401a7f10-->
<!--TEST RESULTS-->
<!--b79a48cb-e166-4613-ab4d-63d0cff9f8c6-->
<!--TEST RESULTS-->
<!--3ffa974e-755a-4e64-98d5-da5dd242cf2d-->
<!--title-->
## CI test results
<!--/title-->
<!--body-->
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/59719">build#59719</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=False.with_iceberg=False.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b03-4737-ab39-025011aad097 | FLAKY | 1/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b08-495d-9134-c14c31748372 | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b09-4ffa-9bd4-c88c14360b8d | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b0a-4e54-ad7e-abfa1e873de0 | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b0a-4f3c-9dd1-7720842e23d1 | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b0b-4f80-9409-ca684e99df6f | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b0c-4a08-ad78-a5b9927f288f | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b0d-452d-b06b-0df371962a12 | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b0d-4d79-a796-1eab1a3534b7 | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b0e-4002-81a5-d9a4337bc18d | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b0e-46ce-a0cf-f2e9d5bfb8d9 | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=True.with_iceberg=False.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b03-4737-ab39-025011aad097 | FLAKY | 1/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=True.with_iceberg=False.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b03-4ef8-b32d-a8a103d705d0 | FLAKY | 1/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=False.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b04-443b-92b0-5cae1d4db2f9 | FLAKY | 1/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b08-495d-9134-c14c31748372 | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b09-4ffa-9bd4-c88c14360b8d | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b0a-4e54-ad7e-abfa1e873de0 | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b0a-4f3c-9dd1-7720842e23d1 | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b0b-4f80-9409-ca684e99df6f | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b0c-4a08-ad78-a5b9927f288f | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b0d-452d-b06b-0df371962a12 | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b0d-4d79-a796-1eab1a3534b7 | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b0e-4002-81a5-d9a4337bc18d | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b0e-46ce-a0cf-f2e9d5bfb8d9 | FAIL | 0/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=True.with_iceberg=False.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b02-4cd7-a3cc-28cd6387060b | FLAKY | 1/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=True.with_iceberg=False.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b03-4ef8-b32d-a8a103d705d0 | FLAKY | 1/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=True.with_iceberg=False.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b04-443b-92b0-5cae1d4db2f9 | FLAKY | 1/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=True.with_iceberg=True.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b08-495d-9134-c14c31748372 | FAIL | 0/1 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=True.with_iceberg=True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/59719#0193bf91-9b0e-4002-81a5-d9a4337bc18d | FLAKY | 1/2 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/63221">build#63221</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.cloud_retention_test.CloudRetentionTest.test_cloud_retention.max_consume_rate_mb=None.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/63221#0195a40f-b917-436d-9d45-ab25f115a54c | FLAKY | 1/2 |
| rptest.tests.compaction_recovery_test.CompactionRecoveryUpgradeTest.test_index_recovery_after_upgrade | ducktape | https://buildkite.com/redpanda/redpanda/builds/63221#0195a3f5-0936-4514-b599-ac3d3f84dbd3 | FLAKY | 1/2 |
| rptest.tests.crash_loop_checks_test.CrashLoopChecksTest.test_crash_report_with_signal.signo=Signals.SIGABRT.signal_shard=1 | ducktape | https://buildkite.com/redpanda/redpanda/builds/63221#0195a3f5-0937-4cd5-992d-08a70c48fff7 | FLAKY | 1/2 |
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_migrated_topic_data_integrity.transfer_leadership=True.params=.cancellation.dir.in.stage.preparing.use_alias.True | ducktape | https://buildkite.com/redpanda/redpanda/builds/63221#0195a3f5-0937-4cd5-992d-08a70c48fff7 | FLAKY | 1/2 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/63291">build#63291</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=False.with_iceberg=True.with_chunked_compaction=True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/63291#0195a902-64b0-40cd-8a45-78c7e7d28eb9 | FLAKY | 2/3 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=True.with_chunked_compaction=False.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/63291#0195a902-64ad-47df-909a-c27287fa1bb2 | FLAKY | 1/4 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=True.with_chunked_compaction=True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/63291#0195a902-64b0-455f-8c9c-ad185e7d26df | FAIL | 0/20 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=True.with_iceberg=False.with_chunked_compaction=True.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/63291#0195a902-64ae-4de8-8eea-f8c95dbfbd74 | FLAKY | 2/3 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=True.with_iceberg=False.with_chunked_compaction=True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/63291#0195a902-64af-40aa-861b-4120a228be9a | FLAKY | 2/3 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/63300">build#63300</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_migrated_topic_data_integrity.transfer_leadership=True.params=.cancellation.dir.in.stage.prepared.use_alias.True | ducktape | https://buildkite.com/redpanda/redpanda/builds/63300#0195a9ef-f922-4b14-9e4a-25c6acd993bf | FLAKY | 1/2 |
| rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_normalize.dataset_type=JSON | ducktape | https://buildkite.com/redpanda/redpanda/builds/63300#0195a9ef-f923-45a8-9b38-188538298f11 | FLAKY | 1/3 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/63568">build#63568</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=False.with_iceberg=True.with_chunked_compaction=True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/63568#0195c81a-5184-4ced-8539-bcc81fb0999b | FAIL | 0/1 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/63683">build#63683</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.cloud_retention_test.CloudRetentionTest.test_cloud_retention.max_consume_rate_mb=None.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/63683#0195ce69-9eaf-4f76-b191-1e71beb4ec31 | FLAKY | 1/2 |
| rptest.tests.compaction_recovery_test.CompactionRecoveryUpgradeTest.test_index_recovery_after_upgrade | ducktape | https://buildkite.com/redpanda/redpanda/builds/63683#0195ce69-9eaf-4f76-b191-1e71beb4ec31 | FLAKY | 1/2 |
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_migrated_topic_data_integrity.transfer_leadership=False.params=.cancellation.dir.in.stage.preparing.use_alias.False | ducktape | https://buildkite.com/redpanda/redpanda/builds/63683#0195ce62-fead-40f1-ba3a-aa450fbef5ca | FLAKY | 1/2 |
| rptest.tests.describe_producers_test.DescribeProducersTest.test_describe_producer_with_tx.include_group_tx=True | ducktape | https://buildkite.com/redpanda/redpanda/builds/63683#0195ce62-feae-428f-9361-fd28fbe4625e | FLAKY | 1/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=False.with_iceberg=True.with_chunked_compaction=False.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/63683#0195ce69-9eaa-452b-a499-cb52ac609e21 | FAIL | 0/1 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=True.with_chunked_compaction=False.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/63683#0195ce69-9eaa-452b-a499-cb52ac609e21 | FAIL | 0/1 |
| rptest.tests.scaling_up_test.ScalingUpTest.test_scaling_up_with_recovered_topic | ducktape | https://buildkite.com/redpanda/redpanda/builds/63683#0195ce69-9eaa-452b-a499-cb52ac609e21 | FLAKY | 1/2 |
| rptest.tests.tiered_storage_pause_test.TestTieredStoragePause.test_safe_pause_resume.allow_gaps_topic_level=None.allow_gaps_cluster_level=True | ducktape | https://buildkite.com/redpanda/redpanda/builds/63683#0195ce69-9eaf-4f76-b191-1e71beb4ec31 | FLAKY | 1/2 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/63711">build#63711</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.cloud_retention_test.CloudRetentionTest.test_cloud_retention.max_consume_rate_mb=20.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/63711#0195d1ab-7c0f-430a-abaa-685e7cd1b619 | FLAKY | 1/3 |
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_migrated_topic_data_integrity.transfer_leadership=True.params=.cancellation.dir.in.stage.preparing.use_alias.False | ducktape | https://buildkite.com/redpanda/redpanda/builds/63711#0195d1bd-ed8f-4ea6-8061-3e00b565702a | FLAKY | 1/4 |
| rptest.tests.write_caching_fi_e2e_test.WriteCachingFailureInjectionE2ETest.test_crash_all_with_consumer_group | ducktape | https://buildkite.com/redpanda/redpanda/builds/63711#0195d1bd-ed8f-4ea6-8061-3e00b565702a | FLAKY | 1/2 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/63733">build#63733</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_migrated_topic_data_integrity.transfer_leadership=False.params=.cancellation.dir.in.stage.preparing.use_alias.False | ducktape | https://buildkite.com/redpanda/redpanda/builds/63733#0195d34b-df4b-4004-bc17-d01accf98415 | FLAKY | 1/2 |
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_migrated_topic_data_integrity.transfer_leadership=True.params=.cancellation.dir.in.stage.preparing.use_alias.False | ducktape | https://buildkite.com/redpanda/redpanda/builds/63733#0195d34b-df4d-486c-8ac0-b19608d7d29a | FLAKY | 1/2 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/63791">build#63791</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.compaction_recovery_test.CompactionRecoveryUpgradeTest.test_index_recovery_after_upgrade | ducktape | https://buildkite.com/redpanda/redpanda/builds/63791#0195d7b5-6ff1-448e-9cc3-e380a73ab227 | FLAKY | 1/2 |
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_migrated_topic_data_integrity.transfer_leadership=True.params=.cancellation.dir.in.stage.preparing.use_alias.True | ducktape | https://buildkite.com/redpanda/redpanda/builds/63791#0195d79b-4687-467d-9edf-2b93feedea6f | FLAKY | 1/4 |
| rptest.tests.partition_balancer_test.PartitionBalancerTest.test_fuzz_admin_ops | ducktape | https://buildkite.com/redpanda/redpanda/builds/63791#0195d7b5-6ff1-448e-9cc3-e380a73ab227 | FLAKY | 1/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=True.with_chunked_compaction=False.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/63791#0195d7b5-6ff2-4a5d-b1dc-f23a545bf037 | FLAKY | 1/2 |
| rptest.tests.scaling_up_test.ScalingUpTest.test_scaling_up_with_recovered_topic | ducktape | https://buildkite.com/redpanda/redpanda/builds/63791#0195d7b5-6ff2-4a5d-b1dc-f23a545bf037 | FLAKY | 1/2 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/64049">build#64049</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| kafka_server_rpfixture.kafka_server_rpfixture | unit | https://buildkite.com/redpanda/redpanda/builds/64049#0195f247-51d6-40c9-bfdd-07542fdb0bc6 | FLAKY | 1/2 |
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_migrated_topic_data_integrity.transfer_leadership=True.params=.cancellation.dir.in.stage.preparing.use_alias.True | ducktape | https://buildkite.com/redpanda/redpanda/builds/64049#0195f289-834d-45a2-8e0e-47fd12741562 | FLAKY | 1/2 |
| rptest.tests.upgrade_test.UpgradeBackToBackTest.test_upgrade_with_all_workloads.single_upgrade=True | ducktape | https://buildkite.com/redpanda/redpanda/builds/64049#0195f289-834e-4d20-b948-ff1c886ce88e | FLAKY | 1/2 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/64110">build#64110</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.cloud_storage_timing_stress_test.CloudStorageTimingStressTest.test_cloud_storage.cleanup_policy=compact.delete | ducktape | https://buildkite.com/redpanda/redpanda/builds/64110#0195f6ba-59a2-4201-ad99-2ff7a909ec84 | FLAKY | 1/2 |
| rptest.tests.maintenance_test.MaintenanceTest.test_maintenance_sticky.use_rpk=True | ducktape | https://buildkite.com/redpanda/redpanda/builds/64110#0195f6cd-2842-4acb-8f40-485b4908039e | FLAKY | 1/2 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=True.with_iceberg=False.with_chunked_compaction=True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64110#0195f6ba-59a4-4cd0-8938-b7896e706c19 | FLAKY | 1/2 |
| rptest.tests.scaling_up_test.ScalingUpTest.test_scaling_up_with_recovered_topic | ducktape | https://buildkite.com/redpanda/redpanda/builds/64110#0195f6ba-59a1-4d9b-b020-5fd2aa788952 | FLAKY | 1/2 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/64155">build#64155</a></summary>


| test_id                                                                                                                                                                                                                    | test_kind   | job_url                                                                                   | test_status   | passed   |
|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_migrated_topic_data_integrity.transfer_leadership=True.params=.cancellation.dir.in.stage.preparing.use_alias.True                                         | ducktape    | https://buildkite.com/redpanda/redpanda/builds/64155#0195faf4-16d5-44a6-802b-ee8a9b574f0f | FLAKY         | 1/4      |
| rptest.tests.datalake.rest_catalog_connection_test.RestCatalogConnectionTest.test_redpanda_connection_to_rest_catalog.cloud_storage_type=CloudStorageType.S3                                                               | ducktape    | https://buildkite.com/redpanda/redpanda/builds/64155#0195faf7-61fa-4dfe-855a-ff1039de108d | FLAKY         | 1/2      |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=False.with_iceberg=False.with_chunked_compaction=False.cloud_storage_type=CloudStorageType.ABS | ducktape    | https://buildkite.com/redpanda/redpanda/builds/64155#0195faf7-61fc-438c-b17a-d518ced6733a | FAIL          | 0/20     |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=False.with_iceberg=False.with_chunked_compaction=True.cloud_storage_type=CloudStorageType.ABS  | ducktape    | https://buildkite.com/redpanda/redpanda/builds/64155#0195faf7-61fa-4dfe-855a-ff1039de108d | FAIL          | 0/20     |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=True.with_iceberg=False.with_chunked_compaction=False.cloud_storage_type=CloudStorageType.ABS  | ducktape    | https://buildkite.com/redpanda/redpanda/builds/64155#0195faf7-61fc-438c-b17a-d518ced6733a | FAIL          | 0/20     |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=False.mixed_versions=True.with_iceberg=False.with_chunked_compaction=True.cloud_storage_type=CloudStorageType.ABS   | ducktape    | https://buildkite.com/redpanda/redpanda/builds/64155#0195faf7-61fa-4dfe-855a-ff1039de108d | FAIL          | 0/20     |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=False.with_chunked_compaction=False.cloud_storage_type=CloudStorageType.ABS  | ducktape    | https://buildkite.com/redpanda/redpanda/builds/64155#0195faf7-61fc-438c-b17a-d518ced6733a | FAIL          | 0/20     |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_iceberg=False.with_chunked_compaction=True.cloud_storage_type=CloudStorageType.ABS   | ducktape    | https://buildkite.com/redpanda/redpanda/builds/64155#0195faf7-61fa-4dfe-855a-ff1039de108d | FAIL          | 0/20     |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=True.with_iceberg=False.with_chunked_compaction=False.cloud_storage_type=CloudStorageType.ABS   | ducktape    | https://buildkite.com/redpanda/redpanda/builds/64155#0195faf7-61fc-438c-b17a-d518ced6733a | FAIL          | 0/20     |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=True.with_iceberg=False.with_chunked_compaction=True.cloud_storage_type=CloudStorageType.ABS    | ducktape    | https://buildkite.com/redpanda/redpanda/builds/64155#0195faf7-61fa-4dfe-855a-ff1039de108d | FAIL          | 0/20     |
</details>
<!--/body-->

dotnwat commented:
@mmaslankaprv got a merge conflict

mmaslankaprv commented:
/ci-repeat 5
release
skip-units
dt-repeat=2
tests/rptest/tests/random_node_operations_test.py

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/63291-->
<!--e1de6706-ca3b-46a8-9307-d0aa5a1174c6-->
<!--title-->
## Retry command for [Build#63291](https://buildkite.com/redpanda/redpanda/builds/63291)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```
/ci-repeat 1
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":1,"enable_failures":true,"mixed_versions":false,"with_chunked_compaction":true,"with_iceberg":true}
```
<!--/body-->

mmaslankaprv commented:
/ci-repeat 1
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":1,"enable_failures":false,"mixed_versions":false,"with_chunked_compaction":true,"with_iceberg":true}

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/63568-->
<!--a75e82bc-f8fc-4694-8540-dc510b892574-->
<!--title-->
## Retry command for [Build#63568](https://buildkite.com/redpanda/redpanda/builds/63568)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```
/ci-repeat 1
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":1,"enable_failures":false,"mixed_versions":false,"with_chunked_compaction":true,"with_iceberg":true}
```
<!--/body-->

mmaslankaprv committed:
tests/rest_catalog: made rest catalog log level configurable

Signed-off-by: Michał Maślanka <michal@redpanda.com>

mmaslankaprv committed:
tests/dl-verifier: de-duplicate consumed messages

The consumer groups in kafka protocol allow consumer to achieve the
'at least once' delivery semantics. As we are disturbing the cluster
while using the `DatalakeVerifier` we must be ready to handle the
duplicates.

Signed-off-by: Michał Maślanka <michal@redpanda.com>

mmaslankaprv committed:
tests/dl-verifier: include topic into logs

When running verifier to more than one topic at a time it is impossible
to distinguish logs from different verifiers.

Signed-off-by: Michał Maślanka <michal@redpanda.com>

mmaslankaprv committed:
dl/tests: tolerate topic not starting from the beginning

Signed-off-by: Michał Maślanka <michal@redpanda.com>

mmaslankaprv committed:
tests/ops_fuzzer: fixed logging create topic op

Signed-off-by: Michał Maślanka <michal@redpanda.com>

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/63683-->
<!--d5c867e7-6cd5-43d2-87ad-f06649d763c7-->
<!--title-->
## Retry command for [Build#63683](https://buildkite.com/redpanda/redpanda/builds/63683)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```
/ci-repeat 1
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":1,"enable_failures":true,"mixed_versions":false,"with_chunked_compaction":false,"with_iceberg":true}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":1,"enable_failures":false,"mixed_versions":false,"with_chunked_compaction":false,"with_iceberg":true}
```
<!--/body-->

mmaslankaprv commented:
/ci-repeat 1
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":1,"enable_failures":false,"mixed_versions":false,"with_chunked_compaction":true,"with_iceberg":true}

mmaslankaprv committed:
tests/admin_ops: tolerate `ms` suffix

In some RPK versions an `ms` suffix was added when rendering topic
configuration properties of duration type. Added a fix to tolerate such
formatting in admin operations fuzzer.

Signed-off-by: Michał Maślanka <michal@redpanda.com>

mmaslankaprv commented:
/ci-repeat 1
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":1,"enable_failures":false,"mixed_versions":false,"with_chunked_compaction":true,"with_iceberg":true}

mmaslankaprv committed:
tests: reduce random node operations test scale

Reducing number of node operations executed during the test will allow
us to reduce the time test is taking to execute

Signed-off-by: Michał Maślanka <michal@redpanda.com>

mmaslankaprv committed:
tests/rno: always enable fast partition movements

Signed-off-by: Michał Maślanka <michal@redpanda.com>

mmaslankaprv committed:
tests/rno: do not query version in non mixed version mode

Signed-off-by: Michał Maślanka <michal@redpanda.com>

mmaslankaprv committed:
tests/node_ops: add partition to the assertion output

Signed-off-by: Michał Maślanka <michal@redpanda.com>

mmaslankaprv commented:
/ci-repeat 1
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":1,"enable_failures":false,"mixed_versions":false,"with_chunked_compaction":true,"with_iceberg":true}

nvartolomei commented on tests/rptest/services/admin_ops_fuzzer.py:
```diff
@@ -61,6 +61,15 @@ def random_string(length):
+def validate_topic_property(current_value: str, expected_value: str,
+                            property_name: str):
+    if (property_name.endswith(".ms")):
+        # RPK may add "ms" suffix to the value
```
bet this is not rpk but a bug of bazel build with which you are testing locally

does this fail in CI? this should't get committed imho

we should get this in https://github.com/redpanda-data/redpanda/pull/25135 instead

nvartolomei commented on tests/rptest/tests/random_node_operations_test.py:
```diff
@@ -464,36 +458,35 @@ def enable_write_caching_testing():
-                consumers_count=self.consumers_count,
-                compaction_enabled=False)
-            fast_producer_consumer.start()
+        # if running with tiered storage create a topic with fast partition
```
is the comment outdated now? we always create the topic without any "ifs"

nvartolomei commented on tests/rptest/tests/random_node_operations_test.py:
```diff
@@ -328,22 +336,24 @@ def maybe_enable_iceberg_for_topic(self, topic_spec: TopicSpec,
             client.alter_topic_config(topic_spec.name,
                                       TopicSpec.PROPERTY_ICEBERG_MODE,
                                       "key_value")
+            client.alter_topic_config(topic_spec.name,
```
I believe you want `__REDPANDA_TEST_DISABLE_BOUNDED_PROPERTY_CHECKS` set if using this. Currently it seems to be set only for "with_chunked_compaction" test.

andrwng replied:
10s is within the bounds though?

nvartolomei commented on tests/rptest/tests/random_node_operations_test.py:
```diff
@@ -120,6 +122,12 @@ def early_exit_hook(self):
 
     def setUp(self):
```
:( unfortunate that we hog on 2 nodes for this which for half of the tests are not even used and this tests runs for quite a bit of time too. Especially in CDT, we waste dollars on idle machines. Do you think we can split the test entrypoint so this does not happen?

I believe you just need to split the annotation. I.e.
```
@cluster(num_nodes=10)
@matrix(...)
@cluster(num_nodes=3)
@matrix(...)
```

This is based on my reading of code from some time ago. Didn't actually test it.

andrwng commented on tests/rptest/tests/datalake/datalake_verifier.py:
```diff
@@ -263,9 +268,13 @@ def _verify_next_message(self, partition, iceberg_offset, iceberg_key):
                 self._expected_compacted_keys.add(iceberg_key)
                 return
             else:
+                if self._tolerate_deleted_messages:
+                    return
```
Is this mainly for retention? I would have thought compacting keys away is handled by _expected_compacted_keys

mmaslankaprv replied:
This is for the case when consumer starts from offset which is > 0, this may happen in case of failures

andrwng commented on tests/rptest/tests/random_node_operations_test.py:
```diff
@@ -140,14 +140,14 @@ def _setup_test_scale(self):
             self.max_partitions = 32
             self.producer_throughput = 20000
-            self.node_operations = 30
+            self.node_operations = 15
```
Curious if this is a general problem with RandomNodeOperationsTest or one that is specific to Iceberg. If just Iceberg, I wonder if it makes sense to only tweak this when Iceberg is enabled?

andrwng reviewed (dismissed):
LGTM for the most part! I do wonder if the number of ops test is worth only doing for Iceberg, given presumably it reduces the coverage of this test?

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/64155-->
<!--923bcc7e-664b-44a2-a021-afeeb815ab82-->
<!--7ebdd6c8-b075-42b4-bbd4-ea2f8fed5bb5-->
<!--title-->
## Retry command for [Build#64155](https://buildkite.com/redpanda/redpanda/builds/64155)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```

/ci-repeat 1
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":2,"enable_failures":false,"mixed_versions":false,"with_chunked_compaction":false,"with_iceberg":false}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":2,"enable_failures":false,"mixed_versions":true,"with_chunked_compaction":false,"with_iceberg":false}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":2,"enable_failures":true,"mixed_versions":false,"with_chunked_compaction":false,"with_iceberg":false}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":2,"enable_failures":true,"mixed_versions":true,"with_chunked_compaction":false,"with_iceberg":false}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":2,"enable_failures":false,"mixed_versions":false,"with_chunked_compaction":true,"with_iceberg":false}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":2,"enable_failures":false,"mixed_versions":true,"with_chunked_compaction":true,"with_iceberg":false}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":2,"enable_failures":true,"mixed_versions":false,"with_chunked_compaction":true,"with_iceberg":false}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":2,"enable_failures":true,"mixed_versions":true,"with_chunked_compaction":true,"with_iceberg":false}
```
<!--/body-->
